services/api/src/routes/users.js

```diff
@@ -35,24 +37,15 @@ router
       data: expandRoles(authUser),
     };
   })
-  .patch(
-    '/me',
-    validateBody({
-      firstName: yd.string(),
-      lastName: yd.string(),
-      timeZone: yd.string(),
-      theme: yd.string(),
-    }),
-    async (ctx) => {
-      const { authUser } = ctx.state;
-      authUser.assign(ctx.request.body);
+  .patch('/me', validateBody(User.getUpdateValidation()), async (ctx) => {
```

Comment:
This is a bad idea no ? better to hardcode what fields can be modified by the user. 
Just trying to see if could use the USer.model validators to validate the phone number

Reply:
no this is correct. access control is handled on model definitions